### ISSUE TITLE
Website is ugly/broken

### ISSUE THREAD

**Issue by klaasg** [12urenloop/docs.12urenloop#12]
- [ ] File/folder structure:
![image](https://user-images.githubusercontent.com/47608311/226390663-8b02b07d-3882-4924-b16d-06ef65f54db0.png)

- [ ] Markdown newlines:
Raw markdown is:
![image](https://user-images.githubusercontent.com/47608311/226392284-254abe56-a295-46d4-a249-dc16e91ca81b.png)
Built HTML is:
![image](https://user-images.githubusercontent.com/47608311/226390926-6a2f2f5a-f1b2-4e6e-9854-f4cec19909ff.png)

**Comment by redfast00** [comment]
Yeah that's my fault, it's not really markdown, but was useful with the scripts

**Comment by FKD13** [comment]
Should still be updated / fixed

**Comment by mcbloch** [comment]
I'm fixing the navigation. Not sure if I will do everything in this issue. Maybe someone can help.
